src/copytrade/copytrade.es6

```diff
@@ -282,10 +286,16 @@ const state = {
       validateToken(scope.searchToken.token)
         .then(tokenUserData => {
           if (!tokenUserData) throw new Error('Invalid token');
-          refreshTraderStats(tokenUserData.loginid, scope.searchToken.token, scope);
-          scope.searchToken.token = '';
-          scope.searchToken.disable = false;
-          updateLocalStorage(scope);
+          refreshTraderStats(tokenUserData.loginid, scope.searchToken.token, scope)
+            .then(() => {
+              scope.searchToken.token = '';
+              scope.searchToken.disable = false;
+              updateLocalStorage(scope);
+            })
+            .catch((e) => {
+              scope.searchToken.disable = false;
+              $.growl.error(e);
```

Comment:
This won't work please check the above growl message for when refresh trading statistics request fails.